worker/runtime/container.go

```diff
@@ -15,6 +15,14 @@ import (
 
 const GraceTimeKey = "garden.grace-time"
 
+type UserNotFoundError struct {
+	User string
+}
+
+func (u UserNotFoundError) Error() string {
+	return fmt.Sprintf("user '%s' not found", u.User)
```

Comment:
Gaurdian has the following error which might be more descriptive in helping the user rectify the issue
```
unable to find user SOME_MISSING_USER: no matching entries in passwd file
```

Reply:
This error is thrown when we call `RootFSManager.LookupUser` and no user is found. The `LookupUser` looks up an user using the `etc/passwd`. However, we thought the Concourse UI facing user does not care about the details of how we find the user, and only that the user could not be found.

We can change it if you feel strongly about it.

Reply:
Added details to message.